components/automate-ui/src/app/pages/+compliance/shared/reporting/stats.service.ts

```diff
@@ -219,9 +219,9 @@ export class StatsService {
      them private? */
   getControlStatus(control): ControlStatus {
     // In some cases, nodes don't have results data.  In these
```

Comment:
nitpick (on the comment): "nodes" -> "controls"

a profile is executed against a node
a profile is a collection of controls
a control is a collection of tests
each test has a result

Reply:
😅 Comment updated.  Thanks for explaining the reasoning!